xml/System.Net.WebSockets/ClientWebSocket.xml

```diff
@@ -369,7 +369,17 @@
       <Docs>
         <param name="uri">The URI of the WebSocket server to connect to.</param>
         <param name="cancellationToken">A cancellation token used to propagate notification that the operation should be canceled.</param>
-        <summary>Connects to a WebSocket server as an asynchronous operation.</summary>
+        <summary>
+			Connects to a WebSocket server as an asynchronous operation.
+			The request has default headers:
+			VersionPolicy = RequestVersionExact
+			For HTTP/2 request:
+			:protocol = websocket
+			For HTTP/1.1 request:
+			Connection = Upgrade
```

Comment:
I think we should follow conventional headers format, like `Connection: Upgrade`